Makefile

```diff
@@ -179,7 +179,7 @@ e2e-tests: e2e-substitutions cluster-templates ## This target should be called f
 		$(CONTAINER_RUNTIME) pull $$image; \
 	done
 	time go test -v -timeout 24h -tags=e2e ./test/e2e/... -args \
-		-ginkgo.v -ginkgo.trace -ginkgo.progress -ginkgo.failFast -ginkgo.noColor=$(GINKGO_NOCOLOR) \
+		-ginkgo.timeout=6h -ginkgo.v -ginkgo.trace -ginkgo.progress -ginkgo.failFast -ginkgo.noColor=$(GINKGO_NOCOLOR) \
```

Comment:
According to @mboukhalfa thats the timeout we have for e2e probably in project-infra. The reason to add this var was because with gingko v2 the default timeout is now 1 hr which is why the PRs were failing previously.

Reply:
Jenkins timeout after 6h, we can use it as default value but keep it parameterized `timeout ?= 6h`